src/console_reporter.cc

```diff
@@ -45,9 +45,15 @@ bool ConsoleReporter::ReportContext(const Context& context) {
                "affected.\n";
 #endif
 
-  int output_width = fprintf(stdout, "%-*s %10s %10s %10s\n",
+  int output_width = fprintf(stdout, "%-*s %10s %10s %10s",
                              static_cast<int>(name_field_width_), "Benchmark",
                              "Time(ns)", "CPU(ns)", "Iterations");
+
+  if (context.benchmark_best_worse_enabled)
+    output_width += fprintf(stdout, " %10s %10s", "Best(ns)" , "Worse(ns)");
```

Comment:
It depend if the user choose the UseRealTIme() or not.
For every State::KeepRunning() I calculate the min&max values (vector for threads), that it will be used in the RunBenchmark() and stored following the logic of the flag use_real_time.